client/blocks/login/two-factor-authentication/verification-code-form.scss

```diff
@@ -100,3 +113,14 @@
 .security-key-form__add-wait-for-key-heading {
 	font-weight: 600;
 }
+
+div.two-factor-auth-enabled:not(.woo) {
+	.login__form-header {
+		width: 352px;
+		margin: 0 auto;
+		padding: 0 24px;
+		text-align: left !important;
+		line-height: 40px !important;
+		text-wrap: pretty;
```

Comment:
Reminder to omit the pretty text wrap from here in favor of a global one. And can we avoid the !important's?

Reply:
Thanks for the reminder. It is fixed in 8e0ca84